semgrep-core/tests/OTHER/errors/unsatisfiable_formula.yaml

```diff
@@ -2,7 +2,6 @@ rules:
 - id: test-template
   patterns:
   - pattern: eval(...)
-  #ERROR: unsatsfiable
```

Comment:
@aryx I was really confused about why the other files in this folder that had error annotations weren't causing tests to fail? I understand why these two were